sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateOperationProperties.cs

```diff
@@ -123,6 +124,12 @@ public bool? CertificateTransparency
         /// </summary>
         public CertificateOperationError Error { get; internal set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the certificate chain preserves its original order.
+        /// The default value is false, which sets the leaf certificate at index 0.
+        /// </summary>
+        public bool? PreserveCertOrder { get; internal set; }
```

Comment:
Why is "Certificate" abbreviated here?   That goes against the guidelines and is inconsistent with other members of this library.    Unless this was explicitly requested by one of the architects, please use "Certificate" for this beta and we can finalize the name in the GA review archboard.